recipes/libtool/all/conanfile.py

```diff
@@ -57,14 +59,22 @@ def _configure_autotools(self):
         conf_args = [
             "--datarootdir={}".format(datarootdir),
             "--prefix={}".format(prefix),
-            "--disable-shared",
-            "--disable-static",
+            "--enable-shared",
```

Comment:
why did you invert it shared and static flags? was it a bug before?

Reply:
Yes, at first I thought these arguments were only there to create the `libtool-ltdl` library.
But while helping @ericLemanissier at https://github.com/madebr/conan-center-index/commit/047754ea6276f8233228943f4f87611fec0a6393#commitcomment-37797414,
I found out that `libtoolize` needs both enabled.
Otherwise it is unable to  create both shared and static libraries.